website/pages/execution.mdx

```diff
@@ -56,3 +60,25 @@ a GraphQLError will be thrown immediately explaining the invalid input.
 `ExecutionResult` represents the result of execution. `data` is the result of
 executing the query, `errors` is null if no errors occurred, and is a
 non-empty array if an error occurred.
+
+### executeSync
+
+```ts
+export function executeSync(
```

Comment:
```suggestion
export function execute(
```

Reply:
No this is executeSync, as the heading sais

Reply:
Yeah, whoops sorry about that!